src/wallet/node_rpc_proxy.cpp

```diff
@@ -322,9 +340,9 @@ std::pair<bool, std::vector<cryptonote::rpc::GET_SERVICE_NODE_BLACKLISTED_KEY_IM
     if (m_service_node_blacklisted_key_images_cached_height != height)
     {
       try {
-        auto res = invoke_json_rpc<rpc::GET_SERVICE_NODE_BLACKLISTED_KEY_IMAGES>({});
+        auto res = m_http_client.json_rpc("get_service_nod_blacklisted_key_images", {});
         m_service_node_blacklisted_key_images_cached_height = height;
-        m_service_node_blacklisted_key_images               = std::move(res.blacklist);
+        m_service_node_blacklisted_key_images               = std::move(res["blacklist"]);
```

Comment:
does std::move do anything here?

Reply:
Yes, it will still move the contained value rather than copying it, which seems fine since `res` isn't used again.